core/src/scene/sceneLoader.cpp

```diff
@@ -528,6 +525,51 @@ Node SceneLoader::mixStyle(const Mixes& mixes) {
     return styleNode;
 }
 
+Mixes SceneLoader::recursiveMixins(Mixes mixIn, const std::string& styleName, Node styles, UniqueStyles& uniqueStyles) {
+    Mixes mixes;
+    mixes.swap(mixIn);
+
+    static std::unordered_set<std::string> mixedStyles;
+
+    auto styleNode = styles[styleName];
+
+    if (Node mixNode = styleNode["mix"]) {
+        if (mixNode.IsScalar()) {
+            auto mixStyleName = mixNode.as<std::string>();
+            mixes.reserve(mixes.size() + 1);
```

Comment:
`reserve` not needed here - `push_back` will do precisely the same thing /nitpick